homeassistant/components/jewish_calendar/sensor.py

```diff
@@ -275,15 +276,18 @@ def get_state(
             # Compute the weekly portion based on the upcoming shabbat.
             return after_tzais_date.upcoming_shabbat.parasha
         if self.entity_description.key == "holiday":
-            self._attrs = {
-                "id": after_shkia_date.holiday_name,
-                "type": after_shkia_date.holiday_type.name,
-                "type_id": after_shkia_date.holiday_type.value,
-            }
-            self._attr_options = [
-                h.description.hebrew.long if self._hebrew else h.description.english
-                for h in htables.HOLIDAYS
-            ]
+            _id = _type = _type_id = ""
+            _holiday_type = after_shkia_date.holiday_type
+            if isinstance(_holiday_type, Enum):
```

Comment:
Should we flip this around and do just one `isinstance` check?

if isinstance(_holiday_type, list):
    ....
else:
    ...

Reply:
Makes sense.
I was thinking explicit is better than implicit,  but then simple is better than complex 😉